docs/index.html

```diff
@@ -86,50 +82,48 @@ <h4>Accions customitzades</h4>
 
 <div class="box">
   <div class="box-header">
-    <h3 class="box-title">Amb els camps ordenables</h3>
+    <h3 class="box-title">With sortable fields</h3>
   </div>
   <div class="box-body">
-     <p> Si volem que els camps siguin ordenables hem passar el paràmetre <code>:sortable</code>code> amb l'atribut. Per exemple: <code>attribute:sortable</code> </p>
-     <p> Ho fariem de la següent manera <code>&#x3C;%= table_for User, @users, options: {attributes: %w( id:sortable type:sortable name:sortable surnames:sortable), actions: {buttons: %w(show edit), premodel: [:bo, :admin]} } %&#x3E;</code></p>
-     <%= table_for User, @users, options: {attributes: %w( id:sortable type:sortable name:sortable surnames:sortable), actions: {buttons: %w(show edit), premodel: [:bo, :admin]} } %>
-
+     <p> If sortered fields are wanted, the parameter <code>:sortable</code> has to be passed along with the attribute. For example: <code>attribute:sortable</code></p>
+     <p> It can be done by the following way <code>&#x3C;%= table_for User, @users, options: {attributes: %w( id:sortable type:sortable name:sortable surnames:sortable), actions: {buttons: %w(show edit), premodel: [:bo, :admin]} } %&#x3E;</code> </p>
   </div>
 </div>
 
 <div class="box">
   <div class="box-header">
-    <h3 class="box-title">Altres opcions</h3>
+    <h3 class="box-title">Other options</h3>
   </div>
   <div class="box-body">
-    <h4>Imatges</h4>
-    <p> Si l'atribut és una imatge i no li especifiquem el tamany de la imatge, defecte es <code>thumb</code>, si volem mostrar un altre tamany, li passararem com a paràmetre de la següent manera. <code>avatar:thumb</code></p>
+    <h4>Images</h4>
+    <p> If the attribute is an image and its size is not specified, is <code>thumb</code> by default. If other size is wanted, it can be passed as a parameter by the following way: <code>avatar:thumb</code></p>
```

Comment:
I'd transltate:
`is <code>thumb</code> by default` to :
<code>thumb</code> is the default value

Reply:
...
The source file had an error `<code>avatar:thumb</code>` should be different from the default, I'd put as an example `<code>avatar:medium</code>`